docs/experimental/pss.md

```diff
@@ -0,0 +1,176 @@
+---
+title: PSS Messaging
+id: pss
+---
+
+Out of the ashes of Ethereum's vision for a leak-proof decentralised anonymous messaging system - Whisper - comes PSS. Swarm provides the ability to send messages that appear to be normal Swarm traffic, but are in fact messages that may be received and decrypted to reveal their content only by the specific nodes they were intended to be received by.
+
+:::danger
+PSS is currently an experimental feature and messages are not currently encrypted! Encryption is coming soon, but for now this feature should be used for evaluation and non-sensitive purposes only. API's may change!
+:::
+
+PSS provides a pub-sub facility that can be used for a variety of tasks. Nodes are able to listen to messages received for a specific topic in their nearest neighbourhood and create messages destined for another neighbourhood which are sent over the network using Swarm's usual data dissemination protocols.
+
+### Enable PSS On Your Node
+
+In order to use the PSS messaging facilities you must pass the relevant configuration flags when starting your node.
+
+```sh
+	make binary && dist/bee start --api-addr=:8080 --debug-api-addr=:6063 --verbosity=5 --password="x" --data-dir=/Users/sig/.bee72365789 --cors-allowed-origins="*" --bootnode="" --p2p-ws-enable --debug-api-enable --p2p-addr=:7073
+```
+
+### Subscribe and Receive Messages
+
+In order to receive PSS messages you must first enable the p2p websockets api when starting your node.
+
+```sh
+	bee start --p2p-ws-enable
+```
+
+Once your Bee node is up and running, you will be able to subscribe to feeds using websockets. For testing, it is useful to use the [websocat](https://docs.rs/crate/websocat/1.0.1) command line utility.
+
+Here we subscribe to the topic `test-topic`
+
+```sh
+websocat ws://localhost:8080/pss/subscribe/test-topic
+```
+
+Our node is now watching for new messages received in it's nearest neighbourhood.
+
+### Send Messages
+
+Messages can be sent simply by sending a `POST` request to the PSS api endpoint.
+
+When sending messages, we must specify the first two bytes of the recipients swarm address, which represents their neighbourhood. In this case, we want to send to a node with address `49d80b28adccd67a09959a601f0e28521aaeb06bf1bf17e9e012e7b597c3d926`, so we must include the 'target' `49d8`.
+
+```sh
+curl -XPOST localhost:8082/pss/send/test/49d8 --data "test2"
```

Comment:
specify recipient public key in query param `recipient`.  Format is hex-encoded public key  in compressed or uncompressed format